lib/spack/spack/test/modules/lmod.py

```diff
@@ -151,6 +152,19 @@ def test_alter_environment(self, modulefile_content, module_configuration):
         assert len([x for x in content if 'setenv("FOO", "foo")' in x]) == 0
         assert len([x for x in content if 'unsetenv("BAR")' in x]) == 0
 
+    def test_prepend_path_separator(self, modulefile_content,
+                                    module_configuration):
+        """Tests modifications to run-time environment."""
+
+        module_configuration('module_path_separator')
+        content = modulefile_content('module-path-separator')
+
+        for line in content:
+            if re.match(r'[a-z]+_path\("COLON"', line) is not None:
```

Comment:
Minor comment on style (not sure if everybody in the project here will agree, so feel free to discard this if you don't like it). This and the next condition can be written:
```python
if re.match(r'[a-z]+_path\("COLON"', line):
    ....
```
without the additional `is not None`. I find this style more readable.